File: Testing/Stdlib Test Code/test_exit_int.py
from dataclasses import dataclass, field

def __ul_sort(arr, n): arr[:n] = sorted(arr[:n])

class __ul_Exception(Exception):
    def __init__(self, type_name, message):
        self.type_name = type_name
        self.message = message
        super().__init__(message)


def validate(x):
    if (x < 0):
        return (-1)
    return x

def main():
    val = validate(10)
    if (val < 0):
        print(0)
        return 1
    print(val)
    return 0


if __name__ == "__main__":
    main()

File: Testing/Stdlib Test Code/test_exit_int.js

function validate(x) {
    if ((x < 0)) {
        return (-1);
    }
    return x;
}

function main() {
    let val = validate(10);
    if ((val < 0)) {
        console.log(0);
        return 1;
    }
    console.log(val);
    return 0;
}

function __ul_sort(arr, n) { const sub = arr.slice(0, n).sort((a, b) => a < b ? -1 : a > b ? 1 : 0); for (let i = 0; i < n; i++) arr[i] = sub[i]; }

main();
